help/en/package/apps/TabbedPreferences.shtml

```diff
@@ -279,7 +279,7 @@ This will let you to limit what users have the ability to control.
 </dl>
 
 The "Apply" button puts the current settings into effect, without
-saving them permanently.  For a connected throttle to use the
+saving them permanently.  For a connected <a href="../jmri/jmrit/withrottle/UserInterface.shtml">WiThrottle</a> to use the
```

Comment:
"throttle" here refers to a client, not the server, and since one client is named "WiThrottle" and others are not, this probably should remain "throttle" (nothing wrong with the link though).